prow/config.yaml

```diff
@@ -176,9 +176,6 @@ presubmits:
           readOnly: true
         - name: cache-ssd
           mountPath: /root/.cache
-        ports:
```

Comment:
I forgot that we have a naive config test that forces this port on any job with the cache-ssd mounted. I'll have to fix the test... 😞